far/FarEng.hlf.m4

```diff
@@ -3416,6 +3416,18 @@ $ #Editor: All matching entries menu#
  #LeftCtrl+0…9#
  Go to the bookmark 0…9.
 
+ #Alt+Left#, #Alt+Right#
+ Scroll all items horizontally.
+
+ #Alt+Shif+Left#, #Alt+Shif+Right#
```

Comment:
Shif**t**, here and there

Reply:
Ouch! Thank you for pointing this out, @johnd0e.

I moved the description to `@MenuCmd`, added description of the existing keys, and of course fixed the typo.